distribution/changelog.txt

```diff
@@ -28,6 +28,7 @@
 - Fix: [#8142] Reliability of mazes and crooked houses can go below 100%.
 - Fix: [#8187] Cannot set land ownership over ride entrances or exits in sandbox mode.
 - Fix: [#8204] Crash when tile element has no surface elements.
+- Fix: [#8212] Crash when opening rides with changed mode and no tracks.
```

Comment:
You probably meant this for other PR?

Reply:
I didn't mean to change this file at all in this PR, I'll undo it.